server.js

```diff
@@ -6,6 +6,7 @@ var express = require('express'),
     passport = require('passport'),
     logger = require('mean-logger'),
     io = require('socket.io');
+require('dotenv').config();
```

Comment:
Nitpick: Let's move this to the top of the file.

Also, it's specified as a `devDependency`. What happens in production?